application/cache/cache.go

```diff
@@ -0,0 +1,233 @@
+package cache
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/go-redis/cache/v8"
```

Comment:
Since our package is named cache and this github.com/go-redis/cache package exports a `cache` package as well we probably need to alias this import. Something like `rediscache "github.com/go-redis/cache/v8"`.  Another option would be to rename our package, but since I think we're wrapping all the redis cache operations with our cache package it might be cleaner just to alias the redis cache package.